openstack/cerebro/ci/test-values.yaml

```diff
@@ -0,0 +1,50 @@
+image:
+  tag: "12345678"
+  registry: my-registry.sap
+
+vernemq:
+  vernemq:
+    image:
+      repository: "my-registry.sap/cerebro/vernemq"
+      tag: "12345678"
+
+optimizationService:
+  envVariables:
+  - name: "PARAMETER_NUMBER"
+    value: 1
+  - name: "PARAMETER_BOOLEAN"
+    value: False
+  - name: "PARAMETER_STRING"
+    value: "Parameter for optimization-service"
+  - name: "ARRAY_OF_STRING_VALUES"
+    value: "[\"value1\", \"value2\"]"
+
+migrationRecommenderService:
+  envVariables:
+  - name: "PARAMETER_STRING"
+    value: "Parameter for migration-recommender-service"
+
+dcStateService:
+  envVariables:
+  - name: "PARAMETER_STRING"
+    value: "Parameter for dc-state-service"
+
+
+secrets:
+  fileSecrets:
+  - metadata:
+      name: my-file-secret
+    data:
+      content: |
+        Contents of a secret file

```

Comment:
I'd add two examples, since you are looping over this it should improve the testing of the range.

Reply:
Added another example for a file secret.